public/docs/ts/latest/glossary.jade

```diff
@@ -649,38 +632,34 @@ a#snake-case
 .l-sub-section
   block snake-case-defn
     :marked
-      The practice of writing compound words or phrases such that each word is separated by an
-      underscore (`_`). This form is also known as **underscore case**.
+      The practice of writing compound words or phrases such that an
+      underscore (`_`) separates one word from the next. This form is also known as **underscore case**.
 
 :marked
   ## Service
 .l-sub-section
   :marked
-    Components are great and all, but what do we do with data or logic that are not associated
-    with a specific view or that we want to share across components? We build services!
+    For data or logic that is not associated
+    with a specific view or that you want to share across components, build services.
 
     Applications often require services such as a hero data service or a logging service.
-    Our components depend on these services to do the heavy lifting.
 
     A service is a class with a focused purpose.
     We often create a service to implement features that are
     independent from any specific view,
-    provide share data or logic across components, or encapsulate external interactions.
+    provide shared data or logic across components, or encapsulate external interactions.
 
-    See the [Services](!{docsLatest}/tutorial/toh-pt4.html) chapter of the tutorial to learn more.
+    For more information, see the [Services](!{docsLatest}/tutorial/toh-pt4.html) page.
```

Comment:
Put "tutorial" in the name, somehow, to indicate that it's part of the tutorial, not a reference/guide page for services.